recipes/shtns/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+SHTNS="shtns-3.3.1-r694"
+cd src
+./configure --enable-python --disable-openmp
```

Comment:
this probably needs a `--prefix="${PREFIX}"`

Reply:
you're correct, this should be explicit, fixed